stable/postgresql/Chart.yaml

```diff
@@ -1,16 +1,19 @@
 name: postgresql
-version: 0.18.1
-appVersion: 9.6.2
-description: Object-relational database management system (ORDBMS) with an emphasis on extensibility and on standards-compliance.
+version: 1.0.0
```

Comment:
What are the steps to upgrade to this version? Per https://github.com/helm/charts/issues/5657 if there are any steps please document them in an "Upgrade" section of the README.

Reply:
+1